lib/mozart_fetcher/http_client.ex

```diff
@@ -33,6 +33,14 @@ defmodule HTTPClient do
     end
   end
 
+  defp calculateRequestHeaders(endpoint) do
+    if String.starts_with?(endpoint, "https://fabl.api.") do
+      [{"accept-encoding", "gzip"}, {"ctx-unwrapped", "1"}]
+    else
+      [{"accept-encoding", "gzip"}]
+    end
+  end
```

Comment:
You could do something like:

`defp request_headers("https://fabl.api."), do: [{"accept-encoding", "gzip"}, {"ctx-unwrapped", "1"}]
defp request_headers(_endpoint), do: [{"accept-encoding", "gzip"}]
`

Then call `request_headers(endpoint)` on line 10.

Reply:
Fast response! Thanks!
This looks better, have changed it to work like this (and Ettore's suggestion below) :)